
Well in soem way X11 libs are missing if ffmep is compiled without
imlib2 support - maintainers could also test their stuff ...

  - Rene Rebe <rene@rocklinux.org>

--- ffmpeg-0.4.8/Makefile	2003-09-28 17:26:39.000000000 +0200
+++ ffmpeg-0.4.8-fixed/Makefile	2004-04-11 22:58:56.000000000 +0200
@@ -14,6 +14,9 @@
 LDFLAGS+=-p
 endif
 
+# ReneR: well the maintainer could also test his ... without imlib2 support
+EXTRALIBS+=-L/usr/X11/lib -lX11
+
 PROG=ffmpeg$(EXESUF)
 PROGTEST=output_example$(EXESUF)
 
